src/auto-clicker.ts

```diff
@@ -22,7 +22,7 @@
 
   function clickBonus() {
     const interactiveElement = targetNode.querySelector<HTMLButtonElement>(
-      '.jwRWhW'
+      '[aria-label="Claim Bonus"]'
```

Comment:
Tested this change locally on latest version of Chrome and Firefox and it worked fine.

Hopefully this will stop the need to update this when Twitch changes their class names for the claim button.

But knowing Twitch they will change the "Claim Bonus" text in 2 months or so.

Reply:
This does only work with the English locale of Twitch. The aria-label is different to each locale.